Makefile

```diff
@@ -82,23 +82,38 @@ man/man1:
 # The install-man make target are for use by debian packaging.
 # The semantics should match autotools make files as dh_make expects it.
 install-man: man/man1
+	mkdir -p $(DESTDIR)/usr/share/man/man1
 	for file in man/man1/* ; do \
 	 	$(INSTALL_FILE) $$file "$(DESTDIR)/usr/share/man/man1" ; done
 
 uninstall-man: man/man1
 	for file in man/man1/* ; do \
 	 	rm "$(DESTDIR)/usr/share/$$file" ; done
+	-rmdir -p $(DESTDIR)/usr/share/man/man1
+
+install-bash-completion:
+	mkdir -p $(DESTDIR)/usr/share/bash-completion/completions
+	$(INSTALL_FILE) config/bash/charm "$(DESTDIR)/usr/share/bash-completion/completions/"
+
+uninstall-bash-completion:
+	rm "$(DESTDIR)/usr/share/bash-completion/completions/charm"
+	-rmdir -p $(DESTDIR)/usr/share/bash-completion/completions
```

Comment:
Whoa there!

Why do you want to remove all completions?  My system has 780 in that directory.  Even we created the directory we can't be sure there isn't other stuff in there.